net/tstun/wrap_test.go

```diff
@@ -729,11 +735,11 @@ func TestPeerCfg_NAT(t *testing.T) {
 					peer2IP:  selfEIP2,
 					subnetIP: selfEIP2,
 				},
-				dnatMap: map[netip.Addr]netip.Addr{
-					selfNativeIP: selfNativeIP,
-					selfEIP1:     selfNativeIP,
-					selfEIP2:     selfNativeIP,
-					subnetIP:     subnetIP,
+				dnat: []dnatTest{
+					{selfNativeIP, selfNativeIP, selfNativeIP},
+					{peer1IP, selfEIP1, selfNativeIP},
+					{peer2IP, selfEIP2, selfNativeIP},
+					{subnetIP, subnetIP, subnetIP}, // TODO
```

Comment:
I left a TODO here (and below) on the `subnetIP` tests; do they look right?

Reply:
updated